Runtime/Scripts/FileOpen.cs

```diff
@@ -35,21 +51,31 @@ void Start()
 
         DrawHTMLOverCanvas javascriptInput = gameObject.AddComponent<DrawHTMLOverCanvas>();
         javascriptInput.SetupInput(fileInputName, fileExtentions, multiSelect);
-        javascriptInput.AlignObjectID(fileInputName);
+
+        // if button is null, no visual element is attached and we should prevent the DrawHTMLOverCanvas from actually
+        // drawing something over the whole canvas. We still need the HTML input element though as that triggers the
+        // file upload dialog in `OpenFile()`
+        javascriptInput.AlignObjectID(fileInputName, button != null);
+#else
+        if (button) button.onClick.AddListener(OpenFile);
```

Comment:
moet er nog een unsubscribe bij?

Reply:
Goede vraag; die was er niet maar ik kijk even of dat niet beter zou zijn

Reply:
Zo te zien niet, omdat het een button betreft die zijn eigen aan/uit staten heeft -en de button op hetzelfde object moet zitten- is het denk ik beter om dit in `Start` te houden ipv onEnable en onDisable te gebruiken